pkgs/development/python-modules/pandas-datareader/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, isPy27
+, pandas
+, lxml
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "pandas-datareader";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "b2cbc1e16a6ab9ff1ed167ae2ea92839beab9a20823bd00bdfb78155fa04f891";
+  };
+
+  # Tests are trying to load data over the network
+  doCheck = false;
```

Comment:
Can we do an importCheck instead?

Reply:
Could you please point me to an example or some documentation?

Reply:
Nevermind, I found it, it's `pythonImportsCheck`